motorApp/MotorSrc/devMotorAsyn.c

```diff
@@ -208,30 +216,19 @@ static void init_controller(struct motorRecord *pmr, asynUser *pasynUser )
     if (initPos)
     {
         double setPos = devSupDialToRaw(pmr, pmr->dval);
-        epicsEventId initEvent = epicsEventCreate( epicsEventEmpty );
-        RTN_STATUS rtnval;
-
-        pPvt->initEvent = initEvent;
 
-        start_trans(pmr);
-        rtnval = build_trans(LOAD_POS, &setPos, pmr);
-        end_trans(pmr);
-        if (rtnval != OK)  {
+        /* Write setPos to the driver */
+        pPvt->pasynUserSync->reason = pPvt->driverReasons[motorPosition];
+        status = pasynFloat64SyncIO->write(pPvt->pasynUserSync, setPos, pasynUser->timeout);
+        if (status) {
             asynPrint(pasynUser, ASYN_TRACE_ERROR,
-                      "devMotorAsyn::init_controller, %s failed to set position to %f\n",
+                      "devMotorAsyn::init_controller, %s failed to set position to %2.6lf\n",
                       pmr->name, setPos );
         } else {
             asynPrint(pasynUser, ASYN_TRACE_FLOW,
-                      "devMotorAsyn::init_controller, %s set position to %f\n",
+                      "devMotorAsyn::init_controller, %s set position to %2.6lf\n",
```

Comment:
%lf is not a correct format for printf().  It should just be %f.  I don't think we should be giving precision and width, we don't know the magnitude of these values.

Reply:
Thanks for the review, @MarkRivers 
I agree that the %f should be the default for a generic code base.
A new version had been pushed.
Unless we introduce a macro/#define and make the configurable, but that is another story.

Reply:
Please see even the version for upstream motor:
https://github.com/epics-modules/motor/pull/167